### PR TITLE
Define the local stack specific configuration at config/test.exs

### PR DESCRIPTION
**Pull request by yumeto**
@cammellos 
The Integration test environment using LOCALSTACK you created is very nice and easy to use.

When setting up my environment variables according to `README.md`, I noticed that some values ​​could be defined in the `config/test.exs`.

If you like it, please merge this.
(this is experimental proposal, feel free to reject!)

### CODE REVIEW

**Comment by cammellos** [comment]
@yumeto thanks for the PR, I think I will pass on this one, I don't mind the changes (`USE_SSL` is more expressive, and `HOSTNAME` as well), but I would like to be `localstack` agnostic, effectively anyone should be able to use localstack/stubbing server/aws, so hardcoding the port or using `USE_LOCALSTACK` would move in the opposite way.

Let me know if you'd rather use different names for the ENV variables, I have picked those for consistency with `ex_aws` but open to suggestion.

Thanks again for all the contributions! By the way, I think the library might need another core contributor, as ben seems quite busy, so effectively I can't send PR myself as it's difficult to get him to approve them. What do you say if I suggest him to add you ( I can't promise anything or even whether is going to reply to the message :) )? I think it would be a good idea, let me know if you are interested.

**Comment by yumeto** [comment]
@cammellos 
I have understood and agreed with your idea that we do not want to depend on LocalStack as any test parameter, including environment variable.
And your original implementation is clear and simple to set up the pseudo IT.

I thought about the essence of the problem.

What we want to specify at starting test is which type (level) of test to be run:
- UT only --- skip all test which requires any extra preparing of environment. This is default test behavior of many libraries.
- Real IT -- connecting to real AWS service, which requires real auth info.
- Pseudo IT -- This level of test is valuable, because we don't need real AWS account. In the point of  view, it is not necessary to pass `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`. As you pointed out, `USE_LOCALSTACK` is not suitable because it doesn't specify the test level. I'm looking for a keyword that directly specify this level.
  - It may be something like `LOCAL_IT` or `PSEUDO_IT`. By specifying this flag, then the test script sets up all of the required steps: ensures the local stub started (by calling `docker-compose up`), changes the destination of service to local and runs the tests, finally terminate local stub service (by calling `docker-compose down`).
  - Is mix task (alias) like `test.it-local` better for the purpose to set up all at once?
  - If we regards `docker-compose up` is not a part of test but a step of preparing of environment, changing the service destination by passing ones like `AWS_SERVICE_HOST` is the very normal. This is your way which now I agree with. This doesn't relate to test level. I think `AWS_SERVICE_HOST` and `AWS_SERVICE_PORT` are better than `MONITORING_*` because later ones are the matters of docker container side.

I can implement all the pattern above, but I would like to hear of your suggestion first.
- Which test level above should be default when `mix test` ? UT or IT?
  - In other words, is it good idea to pass `--exclude integration-test` argument to skip IT? 
- To run the pseudo IT, is it good to preparing and terminating local stubbing server within the test script? For example implementation, `mix test.it-local` runs `docker-compose up`, set the service destinations, run the test and `docker-compose down`.

Finally, I'm very happy to contribute to these libraries and you!
